packages/slider/src/ux-slider.ts

```diff
@@ -98,36 +98,52 @@ export class UxSlider implements UxComponent {
   }
 
   public updateValue(currentMouseX: number) {
-    const normalizedMouseX = currentMouseX - this.element.offsetLeft;
-    const percentValue = normalizedMouseX / this.element.clientWidth;
+    const rect = this.element.getBoundingClientRect();
+    const normalizedMouseX = currentMouseX - rect.x;
+    const percentValue = normalizedMouseX / rect.width;
     const rawValue = ((this.max - this.min) * percentValue) + this.min;
     const numSteps = Math.round((rawValue - this.min) / this.step);
     const steppedValue = this.min + (this.step * numSteps);
 
     this.value = this.boundValue(steppedValue);
   }
 
-  public onTrackMouseDown() {
+  public onTrackMouseDown(e: MouseEvent | TouchEvent) {
     if (this.disabled) {
       return;
     }
 
     this.isActive = true;
 
     const winEvents = new ElementEvents(window as any);
-    const upAction = (e: MouseEvent) => {
+    const upAction = (e: MouseEvent | TouchEvent | FocusEvent) => {
       if (!this.isActive) {
+        winEvents.disposeAll();
         return;
       }
-
-      this.updateValue(e.clientX);
+      if (e instanceof MouseEvent && e.clientX) {
+        this.updateValue(e.clientX);
+      }
+      if (e instanceof TouchEvent && e.touches.length) {
```

Comment:
`e.touches.length > 0`

Reply:
@bigopon 
Fix: https://github.com/platform5/ux/commit/f41e9dd463d29a0a0a66ead4e10b57fe50b31183

Reply:
@ben-girardet is that commit supposed to be in this PR? I see that it's in a different branch.

Reply:
@bigopon yes it belongs to my fork that is published against this PR

Reply:
Oh but I get it... I should have copied the link from the PR: 
https://github.com/aurelia/ux/pull/232/commits/f41e9dd463d29a0a0a66ead4e10b57fe50b31183